diff --git a/qml/main.qml b/qml/main.qml
index ef3fa29..d0bccd8 100644
--- a/qml/main.qml
+++ b/qml/main.qml
@@ -11,7 +11,8 @@ Item {
     id: root
     visible: true
     clip: true
-
+    
+    property var appdataDir: string("/usr/share/cyber-dock")
     property color backgroundColor: Meui.Theme.darkMode ? Qt.rgba(0, 0, 0, 0.1) : Qt.rgba(255, 255, 255, 0.45)
     property color borderColor: Meui.Theme.darkMode ? Qt.rgba(255, 255, 255, 0.1) : Qt.rgba(0, 0, 0, 0.05)
     property real windowRadius: (Settings.direction === DockSettings.Left) ? root.width * 0.3 : root.height * 0.3
@@ -121,7 +122,7 @@ Item {
             implicitWidth: root.iconSize
             implicitHeight: root.iconSize
             enableActivateDot: false
-            iconName: "qrc:/svg/launcher.svg"
+            iconName: appdataDir + "/svg/launcherIcon.svg"
             popupText: qsTr("Launcher")
             onClicked: process.startDetached("cyber-launcher")
             Layout.alignment: Qt.AlignCenter
